### PR TITLE
Reddit Client using Flask

### PR DESCRIPTION
**Pull request by KaiDevrim**

### CODE REVIEW

**Comment by KaiDevrim** [comment]
https://hackclub.slack.com/files/U012U7V3Y6A/F01B1PLUS0J/zoom_0.mp4

File: workshops/reddit_viewer/README.md [hunk 1]
```diff
@@ -0,0 +1,246 @@
+---   
+
+name: 'Make a Reddit Web App'   
+description: 'Using Flask and the PRAW library make a webpage that allows you to browse subreddits.'   
+author: '@Kai'
```
Comment (sampoder) [inline workshops/reddit_viewer/README.md:5]:
```suggestion
author: '@KaiDevrim'   
```

**Review by sampoder — CHANGES_REQUESTED** [review]
Hey @KaiDevrim!

This feels super super close to being mergeable but just a couple of things

- First up, could you make it a bit clearer about the ID part of the `.env` part, students might be a bit confused what their ID is.

- Also, there are some places with really detailed description... but others that lack it. The main one being adding the POST route which doesn't have much explanation.

- Could you clean up the markdown a bit? Main things: syntax highlighting and images on a new line.

- Could you add 3 hacking examples at the bottom?

Once these changes get made, we can probably merge this workshop!! 

Thank you,

Sam :rocket:

**Comment by KaiDevrim** [comment]
@sampoder What do you mean by syntax highlighting? All my multiline python code is highlighted?

**Comment by KaiDevrim** [comment]
sorry, i was using the wrong file lol! i see now

**Pull request closed without merging**